nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -444,6 +447,12 @@ in
       example = "php82";
     };
 
+    enableCompressedAssets = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
```

Comment:
Am I getting it right that we're serving storing the same data compressed in _several_ formats now by default?